docs/getting-started/pingfederate.md

```diff
@@ -0,0 +1,170 @@
+    <img class="assets" src="../../img/logos/tf-logo.svg" alt="Terraform logo" />
+    <span class="caption">
+        <a class="assetlinks" href="https://registry.terraform.io/providers/pingidentity/pingfederate/latest" target=”_blank”>Registry</a>
+    </span>
+</div>
+
+## MINIMUM VIABLE PRODUCT (MVP) Release
+
+As of late January 2024, the PingFederate Terraform provider is an MVP release. **This status indicates that the provider is in active development and is subject to breaking changes. While available in the Terraform Registry, use of this provider is not recommended for production use at this time.**
+
+## Requirements
+
+* Terraform CLI 1.4+
+* A running PingFederate server accessible over HTTPS, or Docker CLI to start one
+* If using Docker to start a PingFederate server, a DevOps license is required - [Register for the DevOps program here](https://devops.pingidentity.com/how-to/devopsRegistration/)
+
+## Start a PingFederate Docker image to be configured
+
+!!! warning "Using an Existing PingFederate Server"
+    If you already have a running PingFederate server that you can reach over HTTPS, you can skip this step. The provider can be used with any PingFederate server.
+
+First, start a PingFederate server using Docker. Your DevOps credentials will be read from the `${HOME}/.pingidentity/config` file. The HTTPS port (default `9999`) must be exposed.  This example starts the product with the `getting-started/pingfederate` server profile, running the latest version of PingFederate from Docker Hub.
+
+```shell
+docker run --name pingfederate_terraform_provider_container \
+  -d -p 9031:9031 \
+  -d -p 9999:9999 \
+  --env-file "${HOME}/.pingidentity/config" \
+  -e SERVER_PROFILE_URL=https://github.com/pingidentity/pingidentity-server-profiles.git \
+  -e SERVER_PROFILE_PATH=getting-started/pingfederate
+  pingidentity/pingfederate:latest
+```
+
+After starting the container, follow the logs until the server becomes available.
+
+```shell
+docker logs -f pingfederate_terraform_provider_container
+```
+
+After you see the following message in the container logs, the server is ready to receive requests from the provider:
+
+```shell
+PingFederate is up
+```
+
+## Accessing the PingFederate API
+
+Gaining access to the API depends on where your PingFederate instance is running.  In order to work with the API, you need to authenticate just as you would for performing
+administrative tasks.  To interact directly with the API in the browser and view the documentation, typically you would browse to the following URL:
+
+`https://<pf_host>:9999/pf-admin-api/api-docs/`
+
+where **<pf_host\>** is the network address of your PingFederate server. This target can be an IP address, a host name, or a fully qualified domain name. It must be reachable from your computer.
+
+If using OIDC, see the [OIDC Authentication](https://docs.pingidentity.com/r/en-us/pingfederate-110/pf_enabling_oidc_based_auth) page for more information on how to connect.
```

Comment:
Might be worth adding that use of OIDC would be for web console access and isn't supported on the TF provider